portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,105 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Arlet's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+  <div id="first_page">
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+    <div id="h1_">
+    <h1>Arlet's Portfolio</h1>
+    </div>    
+
+    <image src="/images/IMG_2016.jpg" width="350" height="350">
+
+    <p>This is my portfolio. I hope you <i>enjoy</i>!</p>
+
+    <div id="intro">
+    <p>My name is Arlet Miranda. I was born and raised by a single mother in San Jose, CA. 
+          I am a sophomore at UC Berkeley. I am currently residing in on-campus dorms. I love running and playing 
+          the bass guitar. Fun facts: My favorite animal are pigs &#128055; and I have a pet hamster named S'mores. </p>
+    </div>
+
+    <p>Click <a href="/hello">here</a>
+        
+         to view content generated by a servlet.</p>
+    <h2>My <a href= "random.html" <strong>pet</strong></a> gallery! &#128057; (Click on 'pet' for a treat!)</h2>
+    
+    <div id="peru_">
```

Comment:
you'll want to use descriptive names of the content, and not the style.
Another thing that might be useful is to use class (id can only be used once in a file)

A useful classname would be "smores-text"